server/test/unit/server/test_logs.py

```diff
@@ -765,8 +763,7 @@ def fake_getLogger(name=None):
         # We should have used the user's setting
         root_logger.setLevel.assert_called_once_with(logging.ERROR)
 
-    @mock.patch('pulp.server.logs.config.config.get',
-                side_effect=ConfigParser.NoOptionError('server', 'log_level'))
+    @mock.patch('pulp.server.logs.config.config.get', return_value='INFO')
```

Comment:
Won't this cause the log setup to get `INFO` for `config.config.get('server', 'log_type')` and therefore not be able to set a proper logger?  I suppose it will just fail back to syslog at that point due to the `VALID_LOGGERS` check. It's probably safe to do.